src/App.tsx

```diff
@@ -15,71 +17,109 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  selectedGoods: string;
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGoods: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
+  get oneGood(): string {
```

Comment:
```suggestion
  getSelectedGood(): string {
```
Actually, there is no need to use this method, you can just take selectedGood from state